content/blog/2019-01-03-ozunconf18.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: "ozunconf18"
+title: Continuing to Grow Community Together at ozunconf, 2018
+authors:
+  - name: Nicholas Tierney
+    url: https://www.njtierney.com
+    twitter: nj_tierney
+# twitterImg: img/blog-images/2019-01-03-ozunconf18/fun-photo.jpg
+date: 2019-01-03
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- packages
+- data-acess
+---
+
+[summary photo]
+
+In late November 2018, we ran the [third annual rOpenSci ozunconf](http://ozunconf18.ropensci.org). This is the sibling rOpenSci unconference, held in Australia. We ran the first ozunconf in 2016, [in Brisbane](http://auunconf.ropensci.org), the second in 2017 [we ran it in Melbourne](http://ozunconf17.ropensci.org).
```

Comment:
You could link to your summary posts for richer info, rather than the websites
https://ropensci.org/blog/2017/10/31/ozunconf2017/
https://ropensci.org/blog/2016/06/16/au-unconf/

Also fyi, you can get all posts from all ozunconfs with https://ropensci.org/tags/ozunconf/, all ozunconf17 posts with https://ropensci.org/tags/ozunconf17/

Reply:
Awesome, thanks! I didn't know about the tags like that, that's really neat!